core/src/test/scala/org/apache/spark/deploy/worker/WorkerSuite.scala

```diff
@@ -243,4 +243,13 @@ class WorkerSuite extends SparkFunSuite with Matchers with BeforeAndAfter {
       ExecutorStateChanged("app1", 0, ExecutorState.EXITED, None, None))
     assert(cleanupCalled.get() == value)
   }
+  test("test  initRegisteredExecutorsDB  ") {
+    val sparkConf = new SparkConf()
+    Utils.loadDefaultSparkProperties(sparkConf)
+    val securityManager = new SecurityManager(sparkConf)
+    sparkConf.set(config.SHUFFLE_SERVICE_DB_ENABLED.key, "true")
+    sparkConf.set(config.SHUFFLE_SERVICE_ENABLED.key, "true")
+    sparkConf.set("spark.local.dir", "/tmp")
+    val externalShuffleService = new ExternalShuffleService(sparkConf, securityManager)
```

Comment:
Does this test case fail without your patch?

Reply:
@dongjoon-hyun  Thanks a lot,I have fixed scalastyle  and removed  the test ablove  yet.
There is a new test to test this patch. 
Kindly review.